plugins/GrapesJsBuilderBundle/Assets/library/js/builder.js

```diff
@@ -85,10 +85,11 @@ function setThemeHtml(theme) {
       }
 
       // If MJML template, generate HTML before save
-      // if (!textareaHtml.val().length && textareaMjml.val().length) {
-      //   builder.mjmlToHtml(textareaMjml, textareaHtml);
-      // }
-      // }
+      if (!textareaHtml.val().length && textareaMjml.val().length) {
+        const builder = new BuilderService(AssetService.getAssetsConfig());
```

Comment:
@dongilbert do you know why this was uncommented and changed? It's now blocking the release of M5.2 as it prevents create new emails with error:
```
builder.js?v5c3b11ed:76 Uncaught TypeError: h.getAssetsConfig is not a function
    at Object.success (builder.js?v5c3b11ed:76:11169)
```

Reply:
I think it started failing after I merged this automated asset rebuild PR: https://github.com/mautic/mautic/pull/14278

Reply:
https://github.com/mautic/mautic/issues/14299 here's the bug report

Reply:
Not sure if this uncommented code is needed, but the builder service should be initialized like this:
```
const assetService = new AssetService();
const builder = new BuilderService(assetService);
```
Do not revive `getAssetsConfig`